src/main/kotlin/io/kuzzle/sdk/controllers/DocumentController.kt

```diff
@@ -94,20 +95,31 @@ class DocumentController(kuzzle: Kuzzle) : BaseController(kuzzle) {
       index: String,
       collection: String,
       searchQuery: ConcurrentHashMap<String, Any?>,
-      waitForRefresh: Boolean? = null): CompletableFuture<ArrayList<String>> {
+      waitForRefresh: Boolean? = null,
+      lang: Lang = Lang.ELASTICSEARCH): CompletableFuture<ArrayList<String>> {
     val query = KuzzleMap().apply {
       put("index", index)
       put("collection", collection)
       put("controller", "document")
       put("action", "deleteByQuery")
       put("body", searchQuery)
       put("waitForRefresh", waitForRefresh)
+      put("lang", lang.lang)
     }
     return kuzzle
         .query(query)
         .thenApplyAsync { response -> (response.result as ConcurrentHashMap<String?, Any?>)["ids"] as ArrayList<String> }
   }
 
+  fun deleteByQuery(
```

Comment:
```suggestion
  @JvmOverloads
  fun deleteByQuery(
```

Reply:
This conflicts with the previous generated overloads